components/index.ts

```diff
@@ -91,6 +88,8 @@ import { liveSplitManager } from "./livesplit/liveSplitManager"
 import { cheapLoadUserData } from "./databaseHandler"
 import { reportRouter } from "./contracts/reportRouting"
 
+loadFlags()
```

Comment:
Why has this been moved, it was purposely one of the first things so you can use flags immediately. We have to be VERY careful that nothing in the imports try to use flags in their global logic.

Reply:
I didn't consciously move it, so it was probably Webstorm. I've moved it back, shouldn't have been an issue though.

Reply:
Yeah, i’ve been meaning to file a YouTrack issue soon